src/components/dashboard/main-content/CourseRecommendations.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import { Button } from '@edx/paragon';
+import { Link } from 'react-router-dom';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import SkillsQuizImage from '../../../assets/images/skills-quiz/skills-quiz.png';
+
+const CourseRecommendations = () => {
+  const {
+    enterpriseConfig: {
+      slug,
+    },
+  } = useContext(AppContext);
+
+  return (
+    <div className="course-recommendations">
+      <h2 className="course-recommendations-title">Get course recommendations for you</h2>
+      <div className="row course-recommendations-details">
+        <div className="col-lg-6 col-sm-12">
+          <p>
+            Take a two-minute quiz to tell us more about the skills and jobs you are interested in,
+            and immediately receive recommendations for the best learning experiences in the catalog for you.
+          </p>
+          <Button
+            as={Link}
+            to={`/${slug}/skills-quiz`}
+            className="btn-brand-primary d-block d-md-inline-block"
+          >
+            Take the Quiz
```

Comment:
I think we discussed that button text here and on the `Find a course` will be same?

Reply:
I think we decided to keep the text same as "Take the quiz" just as on the dashboard, I will ask the question on ticket for clarification and make the change accordingly.

Reply:
I have changed the text to `Recommend courses for me` as suggested by Nathan at https://2u-internal.atlassian.net/browse/ENT-6235?focusedCommentId=630402